contracts/compound/libraries/LibCollateralSwap.sol

```diff
@@ -38,18 +38,18 @@ library LibCollateralSwap {
         address borrowedAsset,
         uint256 borrowedAmount,
         uint256 fee,
+        IComet comet,
         bytes memory data
     ) external {
         // decode payload
         (
         address user,
-        IComet comet,
```

Comment:
Do you see any issues with keeping the `performCollateralSwap` interface the same and just passing the Comet Market address as part of the `data` payload?